crates/c-api/include/wasmtime/store.h

```diff
@@ -11,6 +11,16 @@
 #include <wasi.h>
 #include <wasmtime/error.h>
 
+#if defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__)
+#ifndef WINDOWS
+#define WINDOWS
+#endif // WINDOWS
+#elif __unix__ || __unix || __APPLE__
+#ifndef UNIX
+#define UNIX
+#endif // UNIX
```

Comment:
Could this choose names such as `WASMTIME_UNIX` or `WASMTIME_WINDOWS` to avoid clashing with other projects?

Reply:
Sure, you may just push a change for styling suggestions like that...